Packs/LINENotify/Integrations/LINENotify/README.md

```diff
@@ -0,0 +1,43 @@
+This is the Hello World integration for getting started.
+https://notify-bot.line.me/doc/en/
+This integration was integrated and tested with version xx of LINENotify
```

Comment:
Is there a product version? if so please specify it.
If not remove this line

Reply:
I would put it as version 1.5 for now.  I am considering to add-on other commands later.